diff --git a/node_modules/react-native-spotify-remote/android/build.gradle b/node_modules/react-native-spotify-remote/android/build.gradle
index 381c0cf..8eb7056 100644
--- a/node_modules/react-native-spotify-remote/android/build.gradle
+++ b/node_modules/react-native-spotify-remote/android/build.gradle
@@ -10,17 +10,18 @@
 //   original location:
 //   - https://github.com/facebook/react-native/blob/0.58-stable/local-cli/templates/HelloWorld/android/app/build.gradle
 
-def DEFAULT_COMPILE_SDK_VERSION = 28
-def DEFAULT_BUILD_TOOLS_VERSION = '28.0.3'
-def DEFAULT_MIN_SDK_VERSION = 16
-def DEFAULT_TARGET_SDK_VERSION = 28
+def DEFAULT_COMPILE_SDK_VERSION = 36
+def DEFAULT_BUILD_TOOLS_VERSION = '36.0.0'
+def DEFAULT_MIN_SDK_VERSION = 24
+def DEFAULT_TARGET_SDK_VERSION = 36
+
 
 def safeExtGet(prop, fallback) {
     rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
 }
 
 apply plugin: 'com.android.library'
-apply plugin: 'maven'
+apply plugin: 'maven-publish'
 
 buildscript {
     // The Android Gradle plugin is only required when opening the android folder stand-alone.
@@ -37,16 +38,23 @@ buildscript {
         }
     }
 }
+ext {
+  compileSdkVersion = 36
+  targetSdkVersion = 36
+  minSdkVersion = 24
+  buildToolsVersion = "36.0.0"
+}
 
 apply plugin: 'com.android.library'
-apply plugin: 'maven'
+apply plugin: 'maven-publish'
 
 android {
-    compileSdkVersion safeExtGet('compileSdkVersion', DEFAULT_COMPILE_SDK_VERSION)
-    buildToolsVersion safeExtGet('buildToolsVersion', DEFAULT_BUILD_TOOLS_VERSION)
+    compileSdkVersion 36
+// buildToolsVersion is not needed with AGP 8+; remove it
+
     defaultConfig {
-        minSdkVersion safeExtGet('minSdkVersion', DEFAULT_MIN_SDK_VERSION)
-        targetSdkVersion safeExtGet('targetSdkVersion', DEFAULT_TARGET_SDK_VERSION)
+        minSdkVersion 24
+        targetSdkVersion 36
         versionCode 1
         versionName "1.0"
     }
@@ -86,78 +94,6 @@ dependencies {
     implementation 'com.facebook.react:react-native:+'
 }
 
-def configureReactNativePom(def pom) {
-    def packageJson = new groovy.json.JsonSlurper().parseText(file('../package.json').text)
-
-    pom.project {
-        name packageJson.title
-        artifactId packageJson.name
-        version = packageJson.version
-        group = "com.reactlibrary"
-        description packageJson.description
-        url packageJson.repository.baseUrl
-
-        licenses {
-            license {
-                name packageJson.license
-                url packageJson.repository.baseUrl + '/blob/master/' + packageJson.licenseFilename
-                distribution 'repo'
-            }
-        }
-
-        developers {
-            developer {
-                id packageJson.author.username
-                name packageJson.author.name
-            }
-        }
-    }
-}
-
-afterEvaluate { project ->
-    // some Gradle build hooks ref:
-    // https://www.oreilly.com/library/view/gradle-beyond-the/9781449373801/ch03.html
-    task androidJavadoc(type: Javadoc) {
-        source = android.sourceSets.main.java.srcDirs
-        classpath += files(android.bootClasspath)
-        classpath += files(project.getConfigurations().getByName('compile').asList())
-        include '**/*.java'
-    }
-
-    task androidJavadocJar(type: Jar, dependsOn: androidJavadoc) {
-        classifier = 'javadoc'
-        from androidJavadoc.destinationDir
-    }
-
-    task androidSourcesJar(type: Jar) {
-        classifier = 'sources'
-        from android.sourceSets.main.java.srcDirs
-        include '**/*.java'
-    }
-
-    android.libraryVariants.all { variant ->
-        def name = variant.name.capitalize()
-        def javaCompileTask = variant.javaCompileProvider.get()
-
-        task "jar${name}"(type: Jar, dependsOn: javaCompileTask) {
-            from javaCompileTask.destinationDir
-        }
-    }
-
-    artifacts {
-        archives androidSourcesJar
-        archives androidJavadocJar
-    }
-
-    task installArchives(type: Upload) {
-        configuration = configurations.archives
-        repositories.mavenDeployer {
-            // Deploy to react-native-event-bridge/maven, ready to publish to npm
-            repository url: "file://${projectDir}/../android/maven"
-            configureReactNativePom pom
-        }
-    }
-}
 
 
 
diff --git a/node_modules/react-native-spotify-remote/ios/RNSpotifyRemoteAuth.m b/node_modules/react-native-spotify-remote/ios/RNSpotifyRemoteAuth.m
index 4039277..01384ef 100644
--- a/node_modules/react-native-spotify-remote/ios/RNSpotifyRemoteAuth.m
+++ b/node_modules/react-native-spotify-remote/ios/RNSpotifyRemoteAuth.m
@@ -36,9 +36,9 @@ static RNSpotifyRemoteAuth *sharedInstance = nil;
 }
 
 -(SPTConfiguration*) configuration{
+    if(_apiConfiguration == nil){ _apiConfiguration = [SPTConfiguration configurationWithClientID:@"f95c8effcc63427e8b98c6a92a9d0c17" redirectURL:[NSURL URLWithString:@"flusso://spotify-auth/"]]; }
     return _apiConfiguration;
 }
-
 #pragma mark Singleton Methods
 
 + (instancetype)sharedInstance {
@@ -72,7 +72,7 @@ static RNSpotifyRemoteAuth *sharedInstance = nil;
 }
 
 -(BOOL)isSpotifyInstalled{
-    return _sessionManager != nil && _sessionManager.spotifyAppInstalled;
+    return YES;
 }
 
 #pragma mark URL handling
